### PR TITLE
Fixed #25686 -- Allow routers without "allow_migrate" method

### PR DESCRIPTION
**Pull request by jcugat**
Fixes regression introduced in e2ea30c
Ticket: https://code.djangoproject.com/ticket/25686
Old PR: https://github.com/django/django/pull/5556

### CODE REVIEW

**Comment by charettes** [comment]
Buildbot, add to whitelist.

File: docs/releases/1.8.7.txt [hunk 1]
```diff
@@ -9,4 +9,4 @@ Django 1.8.7 fixes several bugs in 1.8.6.
 Bugfixes
 ========
 
-* ...
+* Fixed a regression where database routers without a ``allow_migrate`` method raised an exception (:ticket:`25686`).
```
Comment (charettes) [inline docs/releases/1.8.7.txt]: Wrap at 79 characters.

**Comment by charettes** [comment]
Do you think you could add a regression test in `tests/multiple_databases/tests.py`?

**Comment by jcugat** [comment]
I'm looking at it now.

**Comment by jcugat** [comment]
@charettes just noticed that I can't fix this in the `stable/1.9.x` branch since `allow_syncdb` has been removed there. I have fixed it (+ added the regression test) in another branch from `stable/1.8.x` here: https://github.com/jcugat/django/tree/ticket-25686-dj18

Should I open another pull request?

**Comment by charettes** [comment]
I see yes, didn't take into account the `syncdb` removal. Please open a new PR based on your new branch and we'll take care of the forward porting. Sorry for the back and forth here.

**Comment by jcugat** [comment]
No problem :) opened #5561

**Pull request closed without merging**